### ISSUE TITLE
Myst - Masterpiece Edition - GOG - ScummVM - program folder empty after installation

### ISSUE THREAD

**Issue by jeffvandyke** [legluondunet/MyLittleLutrisScripts#30]
Installer downloads files from GOG.com properly, selects English language correctly, and the installer finishes quickly and exits with code 0, but browsing the game folder, it looks like nothing at all is there (no spaces in file path).

Here is the logs from failing to launch:
```
Running /home/jeff/.local/share/lutris/runners/scummvm/bin/scummvm --extrapath=/home/jeff/.local/share/lutris/runners/scummvm/share/scummvm --themepath=/home/jeff/.local/share/lutris/runners/scummvm/share/scummvm --aspect-ratio --fullscreen --gfx-mode=3x --path=/home/jeff/Recreation/Games/Lutris/myst-masterpiece-edition myst-win-mohawk
scummvm: Unrecognized game target 'myst-win-mohawk'
Usage: scummvm [OPTIONS]... [GAME]

Try 'scummvm --help' for more options.
Waiting on children
Exit with returncode 1
```

I can provide any other information needed to resolve this issue

**Comment by AndrewKvalheim** [comment]
I just ran into the same problem with Riven, but updating to Lutris 0.5.5 resolved it.

**Comment by legluondunet** [comment]
Just fixed. Please reinstall the game.

**Comment by jeffvandyke** [comment]
Sorry for the delay, finally got around to testing it, great step forward, but now the error from logs is: 
```
Running /home/jeff/.local/share/lutris/runners/scummvm/bin/scummvm --extrapath=/home/jeff/.local/share/lutris/runners/scummvm/share/scummvm --themepath=/home/jeff/.local/share/lutris/runners/scummvm/share/scummvm --aspect-ratio --fullscreen --gfx-mode=3x --path=/home/jeff/Recreation/Games/Lutris/myst-masterpiece-edition myst-win
Initial process has started with pid 694277
Game is considered started.
scummvm: Unrecognized game target 'myst-win'
Usage: scummvm [OPTIONS]... [GAME]

Try 'scummvm --help' for more options.
Game is considered exited.
Initial process has exited.
All children have exited.
Exit with returncode 256
```

The contents of the installation directory (not empty this time, yay!) are:
```
╭─ ~/R/Games/Lutris/myst-masterpiece-edition
╰─ ls -l
total 355632
-rw-r--r--  1 jeff jeff 63780140 Apr  3  2018 channel.dat
-rw-r--r--  1 jeff jeff    28758 Jul 30  2018 channel_french.dat
-rw-r--r--  1 jeff jeff    26560 Jul 30  2018 channel_german.dat
-rw-r--r--  1 jeff jeff    23186 Jul 30  2018 channel_polish.dat
-rw-r--r--  1 jeff jeff    11076 Jul 30  2018 channel_spanish.dat
-rw-r--r--  1 jeff jeff  4870667 Apr  3  2018 credits.dat
-rw-r--r--  1 jeff jeff    80722 Jul 30  2018 credits_french.dat
-rw-r--r--  1 jeff jeff    82336 Jul 30  2018 credits_german.dat
-rw-r--r--  1 jeff jeff  8796556 Apr  3  2018 dunny.dat
drwxr-xr-x  3 jeff jeff     4096 Apr 21 00:14 french
drwxr-xr-x  3 jeff jeff     4096 Apr 21 00:14 german
-rw-r--r--  1 jeff jeff  1849077 Apr  3  2018 help.dat
-rw-r--r--  1 jeff jeff  1864898 Jul 30  2018 help_french.dat
-rw-r--r--  1 jeff jeff  1923844 Jul 30  2018 help_german.dat
-rw-r--r--  1 jeff jeff  2809658 Jul 30  2018 help_polish.dat
-rw-r--r--  1 jeff jeff  3161356 Apr  3  2018 intro.dat
-rw-r--r--  1 jeff jeff     4561 Jun 26  2018 LICENSE.NOTO.txt
-rw-r--r--  1 jeff jeff   592448 Apr  3  2018 manual.pdf
-rw-r--r--  1 jeff jeff 33073173 Apr  3  2018 mechan.dat
-rw-r--r--  1 jeff jeff   133862 Jul 30  2018 mechan_french.dat
-rw-r--r--  1 jeff jeff   136060 Jul 30  2018 mechan_german.dat
-rw-r--r--  1 jeff jeff   134658 Jul 30  2018 mechan_polish.dat
-rw-r--r--  1 jeff jeff    44656 Jul 30  2018 mechan_spanish.dat
-rw-r--r--  1 jeff jeff   284830 Jun 26  2018 menu.dat
-rw-r--r--  1 jeff jeff 74070233 Apr  3  2018 myst.dat
-rw-r--r--  1 jeff jeff  6549364 Jul 30  2018 myst_french.dat
-rw-r--r--  1 jeff jeff  6652814 Jul 30  2018 myst_german.dat
-rw-r--r--  1 jeff jeff  6723446 Jul 30  2018 myst_polish.dat
-rw-r--r--  1 jeff jeff  2571548 Jul 30  2018 myst_spanish.dat
-rw-r--r--  1 jeff jeff   314640 Jun 26  2018 NotoSans-ExtraBold.ttf
drwxr-xr-x  3 jeff jeff     4096 Apr 21 00:14 polish
drwxr-xr-x 10 jeff jeff     4096 Apr 21 00:15 qtw
-rw-r--r--  1 jeff jeff 69538999 Apr  3  2018 selen.dat
-rw-r--r--  1 jeff jeff  2452425 Jul 30  2018 selen_french.dat
-rw-r--r--  1 jeff jeff   349160 Jul 30  2018 selen_polish.dat
-rw-r--r--  1 jeff jeff    61110 Jul 30  2018 selen_spanish.dat
drwxr-xr-x  3 jeff jeff     4096 Apr 21 00:14 spanish
-rw-r--r--  1 jeff jeff 69459412 Apr  3  2018 stone.dat
-rw-r--r--  1 jeff jeff  1250240 Jul 30  2018 stone_french.dat
-rw-r--r--  1 jeff jeff   145814 Jul 30  2018 stone_german.dat
-rw-r--r--  1 jeff jeff   147880 Jul 30  2018 stone_polish.dat
-rw-r--r--  1 jeff jeff    62126 Jul 30  2018 stone_spanish.dat
```

**Comment by legluondunet** [comment]
This time I can not reproduce your issue.
Did you update Lutris to last version 0.5.6?
Could you try to reinstall the ScummVM runner (delete, reinstall).

**Comment by jeffvandyke** [comment]
Alright, tried again, upgraded Lutris to 0.5.6, deleted ScummVM, then tried re-installing the program, but got the same end result as before - it tries to run something called `myst-win`, but that file isn't present.

I did notice: is the `"gog"` field normal in this section from the Myst install script?:
```
    "files": [
        {
            "inno": "http://constexpr.org/innoextract/files/snapshots/innoextract-1.8-dev-2019-01-13/innoextract-1.8-dev-2019-01-13-linux.tar.xz"
        },
        {
            "gog": "N/A:Select the GOG setup file"
        }
    ],
```

**Comment by legluondunet** [comment]
I can finally reproduce your issue, I'll give you info soon...

**Comment by legluondunet** [comment]
It's another issue relative to Lutris: for the moment it is difficult to change or choose language with Lutris ScummVM runner.
The first issue of this thread is resolved, so I close this one and open another issue.
